app/models/concerns/hashed_password_overrides.rb

```diff
@@ -0,0 +1,55 @@
+# In order to perform scrypt calculation of hashed password in a single
+# place for both password and PII encryption, we override
+# a few methods to build the encrypted_password via KeyMaker.
+#
+module HashedPasswordOverrides
+  extend ActiveSupport::Concern
+
+  attr_accessor :hashed_password
+
+  # hashed_password and unlock_password are new methods.
+  def unlock_password(password)
```

Comment:
I would expect to see this defined *below* where it is used in the file (L31 as of right now)